public/app/features/dashboard/components/PanelEditor/state/actions.ts

```diff
@@ -67,6 +67,25 @@ function updateDuplicateLibraryPanels(modifiedPanel: PanelModel, dashboard: Dash
   }
 }
 
+export function skipPanelUpdate(modifiedPanel: PanelModel, panelToUpdate: PanelModel): boolean {
+  // don't update library panels that aren't of the same type
+  if (panelToUpdate.libraryPanel?.uid !== modifiedPanel.libraryPanel!.uid) {
+    return true;
+  }
+
+  // don't update the modifiedPanel twice
+  if (panelToUpdate.id && panelToUpdate.id === modifiedPanel.editSourceId) {
+    return true;
+  }
+
+  // don't update library panels that are repeated
+  if (panelToUpdate.repeatPanelId && panelToUpdate.repeatPanelId === modifiedPanel.editSourceId) {
```

Comment:
Is this a correct assumption for repeats @torkelo ?

Reply:
yes, but you could simply the condition to be just if (panelToUpdate.repeatPanelId)    

So this will skip updating repeated clones, it means that the clones will still be using old settings as the clones are only updated when processRepeats is called. So at the end of the action we should check if the modifiedPanel has a repeat property and if so call dashboard.processRepeats()

Reply:
Thank you , will have a look

Reply:
Added processRepeats and I think it works as expected, great catch!